src/PrestaShopBundle/Controller/Admin/ProductController.php

```diff
@@ -607,8 +607,8 @@ public function formAction($id, Request $request)
         }
 
         $doctrine = $this->getDoctrine()->getManager();
-        $language = empty($languages[0]) ? ['id_lang' => 1, 'id_shop' => 1] : $languages[0];
-        $attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop((int) $language['id_lang'], (int) $language['id_shop']);
+        $languageId = empty($languages[0]) ? 1 : $languages[0]['id_lang'];
+        $attributeGroups = $doctrine->getRepository('PrestaShopBundle:Attribute')->findByLangAndShop((int) $languageId, (int) $this->get('prestashop.adapter.shop.context')->getContextShopID());
```

Comment:
As soon as you switch on a "group" or "all" context, `$this->get('prestashop.adapter.shop.context')->getContextShopID()` returns `null`, translated as `0` with the integer cast.
This makes the SQL request wrong in my opinion, as the attributes are not returned anymore.

Reply:
hi,
thanks, 
I do not really know how to solve this properly, as when regarding theorically, to solve all case , i wonder if : 
- it it a good idea to allow combination creations on a context upper than one Store context , on admin product page, editing a product on Group Context is allready forbidden, 
```
        //If context shop is define to a group shop, disable the form
        if ($shopContext->isShopGroupContext()) {
            return $this->render('@Product/ProductPage/disabled_form_alert.html.twig', ['showContentHeader' => false]);
        }
```

- attributes and attributes values may belong to all shops ? ( this is a breaking change for all the core )

Regards